Lesson_3/home_work3_3.py

```diff
@@ -0,0 +1,10 @@
+friends = ["John", "Marta", "James"]
+enemies = ["John", "Johnatan", "Artur"]
+
+for friend in friends:
+    if friend == "James":
+        print(f"{friend} is the best friend")
```

Comment:
according to task we should not do anything

Reply:
not clear something wrong?